.github/workflows/_update-commit-hash.yml

```diff
@@ -36,7 +36,7 @@ jobs:
       - name: Checkout
         shell: bash
         run: |
-          git clone https://github.com/pytorch/${{ inputs.repo-name }}.git --depth=1 --quiet
+          git clone https://github.com/pytorch/${{ inputs.repo-name }}.git --quiet
```

Comment:
I guess using commit hash from a different branch won't work with a shallow clone (depth=1) and that's why you remove that parameter here. Lucky for us, this only runs once a day, so it's probably